Frontend/ui-common/src/Engineering/Helpers/Utils.js

```diff
@@ -110,6 +113,30 @@ export const getWeeksInMonth = function (year) {
 
 export const getCurrentDay = function (dummy) {
   var date = new Date();
-  return { utcDate: date.toISOString(), date: date.getDate(), shortMonth: date.toLocaleString('default', { month: 'short' }), year: date.getFullYear(), intMonth : date.getMonth(),
+  // var startDate = new Date(date.getFullYear(), date.getMonth(), date.getDate());
+  return { day: getISODate(date), date: date.getDate(), shortMonth: date.toLocaleString('default', { month: 'short' }), year: date.getFullYear(), intMonth : date.getMonth(),
            isInRange : false, isStart: false , isEnd: false }
 }
+
+export const compareDates = function (day1){
+  return function (day2) {
+    if (day1 == "" || day2 == "") return "INV";
+    let dateOne = new Date(day1).toISOString();
+    let dateTwo = new Date(day2).toISOString();
+    if(dateOne == dateTwo) return "EQ";
+    else if(dateOne < dateTwo) return "LT";
+    else if(dateOne > dateTwo) return "GT";
+    else "INV";
+  };
+  
+}
+
+export const differenceBetweenTwoDay = function (str1) {
+  return function (str2) {
+    var curr1 = new Date(str1);
+    var departure = new Date(str2);
+    var diff =(curr1.getTime() - departure.getTime())/ 1000;
```

Comment:
Can you update the variable name on the basis of whether it return difference in seconds, minutes or Hours? 
eg - differenceInSeconds

Reply:
updated to `differenceBetweenTwoDayInSeconds`